client/state/sites/test/selectors.js

```diff
@@ -1184,4 +1194,675 @@ describe( 'selectors', () => {
 			expect( frontPageType ).to.eql( 'page' );
 		} );
 	} );
+
+	describe( '#canJetpackSiteManage()', () => {
+		it( 'it should return `null` for a non-existing site', () => {
+			const state = {
+				sites: {
+					items: {}
+				}
+			};
+			let siteId;
+
+			const canManage = canJetpackSiteManage( state, siteId );
+			expect( canManage ).to.equal( null );
+		} );
+
+		it( 'it should return `null` for a non jetpack site', () => {
+			const siteId = 77203074;
+
+			const state = {
+				sites: {
+					items: {
+						77203074: {
```

Comment:
I wouldn't try to be too clever in tests. The more static the data we give as input the better I think